Woodall_Robert/assignments/ruby_tdd/apple_tree/apple_tree_spec.rb

```diff
@@ -0,0 +1,57 @@
+require_relative "apple_tree"
+
+RSpec.describe AppleTree do
+	before do
+		@apple_tree = AppleTree.new
+	end
+	
+	it "should have a height and age" do
+		expected_height = 10
+		expected_age = 8
+		
+		@apple_tree.height = expected_height
+		@apple_tree.age = expected_age
+		
+		expect(@apple_tree.height).to eq(expected_height)
+		expect(@apple_tree.age).to eq(expected_age)
+	end
+	
+	it "should have a count of how many apples are on the tree" do
+		expect(@apple_tree.apple_count).to eq(0)
```

Comment:
This will fail with another test that manipulates the apple count. You can use a new apple tree just for this test to make sure it gets created with a 0 apple count